### PR TITLE
add subscores for cbcl

### PR DESCRIPTION
**Pull request by yibeichan**
based on the following scoring criteria, I added subscores for abcd_cbcl01
```
8 syndrome scales:

cbcl_anxious_raw: [14, 29, 30, 31, 32, 33, 35, 45, 50, 52, 71, 91, 112]
cbcl_withdrawn_raw: [5, 42, 65, 69, 75, 102, 103, 111]
cbcl_somatic_c_raw: [47, 49, 51, 54, 56a-g]
cbcl_social_p_raw: [11, 12, 25, 27, 34, 36, 38, 48, 62, 64, 79]
cbcl_thought_raw: [9, 18, 40, 46, 58, 59, 60, 66, 70, 76, 83, 84, 85, 92, 100]
cbcl_attention_raw: [1, 4, 8, 10, 13, 17, 41, 61, 78, 80]
cbcl_rulebreak_raw: [2, 26, 28, 39, 43, 63, 67, 72, 73, 81, 82, 90, 96, 99, 101, 105, 106]
cbcl_aggresive_raw: [3, 16, 19, 20, 21, 22, 23, 37, 57, 68, 86, 87, 88, 89, 94, 95, 97, 104]

(NOT IN NDA) 
cbcl_other_raw: [6, 7, 15, 24, 44, 53, 55, 56h, 74, 77, 93, 98, 107, 108, 109, 110, 113]
Broader problem scales:

cbcl_internal_raw: [anxious_depressed, withdrawn_depressed, somatic_complaints]
cbcl_external_raw: [rule_breaking, aggressive_behavior]
cbcl_total_raw: sum ALL 8 syndrome scale totals + cbcl_other_raw items detailed above 
Cbcl/6-18 DSM-Oriented Scales (6 scales)

cbcl_deresspr_raw: [5, 14, 18, 24, 35, 52, 54, 76, 77, 91, 100, 102, 103]
cbcl_anxiety_raw: [11, 29, 30, 31, 45, 47, 50, 71, 112]
cbcl_somatic_p_raw: [56a-g]
cbcl_adhd_raw: [4, 8, 10, 41, 78, 93, 104]
cbcl_oppositional_raw: [3, 22, 23, 86, 95]
cbcl_conduct_raw: [15, 16, 21, 26, 28, 37, 39, 43, 57, 67, 72, 81, 82, 90, 97, 101, 106]
```
some notes:
- `cbcl_deresspr_raw` this name doesn't make much sense to me, so I change it to `cbcl_depress_raw`
- not sure what `Cbcl/6-18 DSM-Oriented Scales (6 scales)` means for subsocring
- the current `abcd_cbcl01_schema` is version `0.0.1/rc1`. should I change it to `0.0.4/rc4`? If we change the version here, should we do it for all old version schemas?
- for the above 18 scores, some of them are in NDA, and some are not. I'm wondering whether we need to add some notes/descriptions in the json file to indicate whether this subscore is in NDA.

### CODE REVIEW

**Comment by satra** [comment]
@yibeichan - should this be adjusted for rc4 instead of master?

**Comment by yibeichan** [comment]
okay, let me check it and get back here this weekend!

**Comment by yibeichan** [comment]
@satra i change it to enh/rc4 and validated with the latest model. we can merge this one and then I'll create a new PR for the new ABCD conversion?

File: activities/abcd_cbcl01/items/cbcl_adhd_raw [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "@context": "https://raw.githubusercontent.com/ReproNim/reproschema/1.0.0-rc1/contexts/generic",
+    "@type": "reproschema:Field",
+    "@id": "cbcl_adhd_raw",
+    "prefLabel": "cbcl_adhd_raw",
+    "schemaVersion": "1.0.0-rc1",
```
Comment (satra) [inline activities/abcd_cbcl01/items/cbcl_adhd_raw:6]: can we change schemaVersion here and in context? and in all the files.

Comment (yibeichan) [reply activities/abcd_cbcl01/items/cbcl_adhd_raw]: done! (sorry i don't know why i can't get notifications from review

**Pull request merged**